### PR TITLE
Tweak Functions error messages to be consistent with Android and C++.

### PR DESCRIPTION
**Pull request by bklimt**
b/123953864

### CODE REVIEW

File: Functions/Example/IntegrationTests/FIRIntegrationTests.m [hunk 1]
```diff
@@ -139,6 +139,7 @@ - (void)testMissingResult {
       callWithCompletion:^(FIRHTTPSCallableResult *_Nullable result, NSError *_Nullable error) {
         XCTAssertNotNil(error);
         XCTAssertEqual(FIRFunctionsErrorCodeInternal, error.code);
+          XCTAssertEqualObjects(@"Response is missing data field.", error.localizedDescription);
```
Comment (paulb777) [inline Functions/Example/IntegrationTests/FIRIntegrationTests.m]: Fix indentation and other travis errors

Comment (bklimt) [reply Functions/Example/IntegrationTests/FIRIntegrationTests.m]: Done.

**Comment by bklimt** [comment]
PTAL

**Comment by paulb777** [comment]
Now test failures running on travis - https://travis-ci.org/firebase/firebase-ios-sdk/jobs/521028378

**Comment by bklimt** [comment]
Arg, it said all the checks were successful when I sent it. I guess it ran this later. Sorry about that.

This looks like a problem with the test runner. It didn't install any of the backend functions needed for the test. I'm not sure why. I'll have to look into it more tomorrow. :-/

**Comment by paulb777** [comment]
Looking at https://travis-ci.org/firebase/firebase-ios-sdk/builds, the Functions tests started failing on master sometime after 2PM PT yesterday.

**Comment by paulb777** [comment]
Created #2851

**Comment by bklimt** [comment]
They rolled back the broken release of the functions-emulator, so now the tests are passing again.

**Pull request merged**